README.md

```diff
@@ -113,6 +113,22 @@ The Criteria Section of the config file is simply named "Criteria" (without the
 - 	Filters: a list of the names of the filters you want to apply
 - 	Start Date: a date filter in the format YYYY-MM-DD which will exclude issues with resolved dates before the provided date (optional)
 - 	End Date: a date filter in the format YYYY-MM-DD which will include issues with resolved dates before the provided date (optional)
+-   BlockedAttributes: This optional feature will add and value the ‘Days Blocked’ column to the CSV extract with the number of days blocked for each issue. 
```

Comment:
This is not adding a "Days Blocked" column to my output.csv file, is it only going to add the column if there is data returned, or should it at least return 0 for every issue (with or without blocked data)?

Reply:
Okay, I'll look into make the change and defaulting.